.vscode/launch.json

```diff
@@ -17,7 +17,8 @@
                 "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
                 "SDV_HVACSERVICE_ADDRESS": "grpc://localhost:50052",
                 "SDV_SEATSERVICE_ADDRESS": "grpc://localhost:50051",
-                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555"
+                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555",
+                "PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION": "python"
```

Comment:
Why is that needed?

Reply:
Ok - had a fast googling about it: As far as I understood another solution would be to increase protobuf to version 3.20.x (see https://github.com/protocolbuffers/protobuf/issues/10051).
(In C++ SDK we use protobuf 3.21.9)

Reply:
grpcio is required protobuf >4

Reply:
My question related to the env var PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION and why we need that.
In my "self-reply" above I mentioned that people in the linked thread said, that this variable isn't required for protobuf > 3.20.x

Reply:
@BjoernAtBosch , advice is to downgrade protobuf to 3.20 and low....
![image](https://github.com/eclipse-velocitas/vehicle-app-python-sdk/assets/97447206/91b60c3d-4253-4821-91f3-bd8cc7b6043e)

In this case it will be not possible to update grpcio which include security fixes for 6 vulnerabilities. grpcio >= 1.53 is required protobuf < 4

It means that only env variable is suitable in our case.

Reply:
Ah - yes, you're right. I was puzzled by this somewhat confusing versioning scheme of protobuf.